core/src/main/java/tc/oc/pgm/listeners/ChatDispatcher.java

```diff
@@ -302,6 +322,29 @@ public void send(
       String format,
       Predicate<MatchPlayer> filter,
       @Nullable SettingValue type) {
+
+    Set<Player> recipients =
+        match.getPlayers().stream()
+            .filter(filter)
+            .map(MatchPlayer::getBukkit)
+            .collect(Collectors.toSet());
+
+    // Call a chatevent for each message, will listen for any changes done by anything other than
+    // PGM and adjust if necessary
+    if (sender != null) {
+      AsyncPlayerChatEvent event =
+          new AsyncPlayerChatEvent(false, sender.getBukkit(), message, recipients);
+
+      messageCache.put(event, ""); // The value in the Cache is not used for anything
+
+      match.callEvent(event);
+
+      if (!event.getMessage().equals(message)) message = event.getMessage();
+      if (!event.getRecipients().equals(recipients)) recipients = event.getRecipients();
+      if (!event.getFormat().equals(format)) format = event.getFormat();
```

Comment:
Figured out the issue.

You’re replacing `format` with the format provided by `AsyncPlayerChatEvent`, which in return is following the String [format syntax](https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html#syntax).

I would suggest simply omitting this check, as `send()` is provided a specific format to PGM (team/global/admin) and I don’t believe we need to care about changes made for that. 

Tested this change locally and all works well.

Reply:
Ah, great catch. I will remove the check. Thanks!